tests/plugins/__snapshots__/renderer.test.ts.snap

```diff
@@ -115,13 +115,18 @@ Array [
   "context.moveTo(0, 989)",
   "context.lineTo(952, 989)",
   "context.stroke()",
+  "context.save()",
+  "context.beginPath()",
+  "context.rect(0, 0, 952, 974)",
+  "context.clip()",
```

Comment:
I would probably do a test that we clip what we dran on the frozen panes. It shouldn't be _too_ annoying

Reply:
I added some tests related to the renderer and he new getter 'getRect'. Not sure about the naming but 'getRenderingRect' was definitely wrong. it's more used as a computational tool.